lib/linux.js

```diff
@@ -38,7 +38,17 @@ const trash = async (filePath, trashPaths) => {
 };
 
 export default async function linux(paths) {
-	const mountPointMap = new Map(procfs.processMountinfo().map(info => [info.devId, info.mountPoint]));
+	// • filter irrelevant
+	// • “first one wins” if id comes along twice
+	const mountPointMap = new Map();
+
+	for (const m of Object.values(procfs.processMountinfo())
```

Comment:
Don't use single-word variable name.

Reply:
Single-character-variables? (in extremly small scopes…), OK, can do :)